seed/views/v3/properties.py

```diff
@@ -1758,6 +1758,63 @@ def delete_inventory_document(self, request, pk):
         doc_file.delete()
         return JsonResponse({"status": "success"})
 
+    @swagger_auto_schema(
+        manual_parameters=[
+            AutoSchemaHelper.query_org_id_field(),
+            AutoSchemaHelper.query_integer_field(
+                name="access_leevl_instance_id", required=True, description="Access Level Instance to move properties to"
+            ),
+        ],
+        request_body=AutoSchemaHelper.schema_factory(
+            {"property_view_ids": ["integer"]},
+            required=["property_view_ids"],
+            description="A list of property view ids to move between alis",
+        ),
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @action(detail=False, methods=["POST"])
+    @has_perm_class("can_modify_data")
+    def move_properties_to(self, request):
+        """
+        Move properties to a different ali
+        """
+        org_id = self.get_organization(request)
+        ids = request.data.get("property_view_ids", [])
+        ali_id = request.query_params.get("access_level_instance_id", None)
+
+        if not ali_id:
+            return JsonResponse({"status": "error", "message": "Target ALI not found"}, status=status.HTTP_404_NOT_FOUND)
+
+        # filter ids based on request user's ali
+        ali = AccessLevelInstance.objects.get(pk=request.access_level_instance_id)
```

Comment:
It might be worth checking the user has access to this ali.

You should be able to do this with a decorator like this: https://github.com/SEED-platform/seed/blob/4cf3e362f14059faff77aef1252f66a61e9cc266/seed/views/v3/analyses.py#L80

Reply:
Oh, yes, that is much more straightforward than what I am doing.  Thank you.  I need to move the access_level_instance_id into the post body for this, but that is not a big deal.

Reply:
And that is now done.  Removed the confusing bits with the lft__gte filter in favor of the decorator.  Thanks, Hannah!

Reply:
yay! for future reference, to ensure an ALI is within ones subtree, the lft must be greater *and* the rgt must be less.

Take for example the tree below. If we only compare lft, it would seem that Duodenal was a child of gastric.
![image](https://github.com/user-attachments/assets/a3615a1e-9173-4906-b314-39d571eda0ec)

Reply:
Thanks, that helps make it clear.